docs/devguide/newdashboards.asciidoc

```diff
@@ -137,136 +128,56 @@ ES_URL="http://192.168.3.206:9200" make import-dashboards
+multiple Beats. The dashboards for each Beat are placed under a separate directory with the name of the Beat.
 
-*`-i <elasticsearch_index>`*::
+*`setup.dashboards.index <elasticsearch_index>`*::
 You should only use this option if you want to change the index pattern name that's used by default. For example, if the
 default is `metricbeat-*`, you can change it to `custombeat-*`.
 
-*`-insecure`*::
-If specified, "insecure" SSL connections are allowed.
-
-*`-k <kibana_index>`*::
-The Elasticsearch index pattern where Kibana saves its configuration. The default value is `.kibana`.
-
-*`-key <client_key>`*::
-The client certificate key. The key must be in PEM format.
-
-*`-only-dashboards`*::
-If specified, then only the dashboards, along with their visualizations and searches, are imported. The index pattern is
-not imported. By default, this is false.
-
-*`-only-index`*::
-If specified, then only the index pattern is imported. The dashboards, along with their visualizations and searches, are not imported. By default, this is false.
-
-*`-pass <password>`*::
-The password for authenticating the connection to Elasticsearch by using Basic Authentication. By default no username and password are used.
-
-*`-snapshot`*::
-Using `-snapshot` will import the snapshot dashboards build for the current version. This is mainly useful when running a snapshot Beat build for testing purpose.
-+
-NOTE: When using `-snapshot`, `-url` will be ignored.
-
-*`-url <zip_url>`*::
-Zip archive with the dashboards, available online. The archive can contain Kibana dashboards for a single Beat or for
-multiple Beats.
-
-*`-user <username>`*::
-The username for authenticating the connection to Elasticsearch by using Basic Authentication. By default no username and password are used.
-
-
-[[dashboards-archive-structure]]
-==== Structure of the Dashboards Archive
-
-The zip archive contains dashboards for at least one Beat. The index pattern, dashboards, visualizations and searches
-are available in a separate directory for each Beat, having the name of the Beat. For example the official zip archive (beats-dashboards-{stack-version}) has the following structure:
-
-[source,shell]
--------------------------
-  metricbeat/
-    dashboard/
-    search/
-    visualization/
-    index-pattern/
-  packetbeat/
-    dashboard/
-    search/
-    visualization/
-    index-pattern/
-  filebeat/
-    index-pattern/
-  winlogbeat/
-    dashboard/
-    search/
-    visualization/
-    index-pattern/
-------------------------
-
 
 [[build-dashboards]]
 === Building Your Own Beat Dashboards
 
-For visualizing the dashboards of a Beat in Kibana you need to have configured:
+NOTE:: If you want to modify a dashboard that comes with a Beat, it's better to modify a copy of the dashboard as the Beat overwrites the dashboards during the setup phase in order to have the latest version. For duplicating a dashboard, just use the `Clone` button from the top of the page. 
```

Comment:
Change to: "...it's better to modify a copy of the dashboard because the Beat overwrites..."

Reply:
done